docs/release-source/release/utils.md

```diff
@@ -30,7 +30,17 @@ Restores supplied method
 
 #### `sinon.createStubInstance(constructor);`
 
-Creates a new object with the given function as the protoype and stubs all implemented functions.
```

Comment:
Not sure if this is an improvement. A class IS A function along with its prototype. It's not either/or.  Might help the ones that don't know the basics, but still...